uk.json

```diff
@@ -304,42 +304,42 @@
 			"mfa-verification-failed": "Код двофакторної автентифікації невірний, будь ласка, перевірте ваш застосунок автентифікації ще раз."
 		},
 		"core-plugin": {
-			"plugin-list": "Вмонтовані плагіни",
+			"plugin-list": "Список додатків",
 			"option-search-plugin": "Шукати вмонтовані плагіни",
-			"option-search-plugin-description": "Пошук плагінів за назвою або описом.",
-			"placeholder-search-plugin": "Пошук плагінів..."
+			"option-search-plugin-description": "Пошук додатків за назвою або описом.",
+			"placeholder-search-plugin": "Пошук додатків..."
 		},
 		"third-party-plugin": {
-			"name": "Сторонні плагіни",
+			"name": "Сторонній додаток",
 			"option-restricted-mode": "Обмежений режим",
-			"option-restricted-mode-description": "Обмежений режим ВИМКНЕНО. Увімкніть, щоб вимкнути усі сторонні плагіни.",
+			"option-restricted-mode-description": "Обмежений режим ВИМКНЕНО. Увімкніть, щоб вимкнути усі додатки спільноти.",
 			"button-turn-on": "Увімкнути",
-			"option-browse-community-plugins": "Плагіни спільноти",
-			"option-browse-community-plugins-description": "Огляд і встановлення сторонніх плагінів, створених нашим приголомшливою спільнотою.",
+			"option-browse-community-plugins": "Додатки спільноти",
+			"option-browse-community-plugins-description": "Огляд і встановлення сторонніх додатків, створених нашою дивовижною спільнотою.",
 			"button-browse": "Огляд",
 			"label-exit-restricted-mode": "Вийти з Обмеженого режиму",
-			"label-exit-restricted-mode-description-1": "Плагіни спільноти, як будь-які інші застосунки, що ви завантажуєте, потенційно можуть завдати шкоди вашим файлам, та можуть бути небезпечними.",
-			"label-exit-restricted-mode-description-2": "Безпечність плагінів важлива для нас. Ось що ми робимо:",
-			"label-exit-restricted-mode-description-3": "Незважаючи на наші зусилля, існує невелика ймовірність того, що плагін спільноти може працювати неправильно.",
+			"label-exit-restricted-mode-description-1": "Додатки спільноти, як будь-які інші застосунки, що ви завантажуєте, потенційно можуть завдати шкоди вашим файлам та можуть бути небезпечними.",
+			"label-exit-restricted-mode-description-2": "Безпека додатків важлива для нас. Ось що ми робимо:",
+			"label-exit-restricted-mode-description-3": "Попри наші зусилля, існує невелика ймовірність того, що додаток спільноти може працювати неправильно.",
 			"label-code-review": "Початковий огляд коду",
-			"label-code-review-desc": "Плагіни проходять первинну перевірку коду нашою командою, перш ніж вони з’являться в магазині.",
+			"label-code-review-desc": "Додатки проходять первинну перевірку коду нашою командою, перш ніж вони з’являться в магазині.",
```

Comment:
In the Ukrainian language "додаток" means literally "add-on", not "application".

Despite Google Play continues to use "додаток" as an "application", other apps and stores use "застосунок" (literal translation of "application") or "програма" ("program").

Examples:
1. Government ID service ["Diia"](https://apps.apple.com/us/app/%D0%B4%D1%96%D1%8F/id1489717872) - "застосунок"
2. [Telegram](https://translations.telegram.org/uk/android/settings/Application), one of the most popular messengers in Ukraine - "застосунок"
3. [App Store](https://support.apple.com/uk-ua/guide/iphone/iph87abad19a/17.0/ios/17.0) - "програма"

The old version, "плагін", is a transliteration of "plugin", and therefore not intuitive for all Ukrainian speakers. Because of this, "додаток" is the most inclusive term for "plugin".